mantidproject/mantid
# New parameters for D2B

**Description of work.**
Adds new instrument parameters and definition with better measured pixel size for D2B for the cycle 191.



**To test:**
Check that valid-from dates match the file names.
LoadEmptyInstrument with D2B, check that the newest IDF and IPFs are loaded.
(For this click on the arrow next to the workspace name, and it's in the text below).



*There is no associated issue, this is a periodic addition initiated by the scientists.*




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
